packages/framework-provider-local-infrastructure/package.json

```diff
@@ -38,7 +38,7 @@
     "clean": "rimraf ./dist tsconfig.tsbuildinfo",
     "prepack": "tsc -b tsconfig.json",
     "test:provider-local-infrastructure": "npm run test",
-    "test": "BOOSTER_ENV=test nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
+    "test": "set BOOSTER_ENV=test & nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
```

Comment:
we need to find a way to only have these changes in place for windows, and not on iOS.

Reply:
cross-env instead of set and & seems to work for both

Reply:
never mind, failed on local test